assets/app/views/logs/chromeless-pod-log.html

```diff
@@ -0,0 +1,20 @@
+<div column flex class="content">
+  <div flex class="log-chromeless">
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="largeLog" class="log-size-warning">
+      <span class="pficon pficon-info"></span>
+      Only the previous {{options.tailLines || 1000}} log lines and new log
```

Comment:
No, it's both. Default options are set in `logViewer.js`, but not visible to the view because I didn't want to change the value passed in to the directive.